modules/uhttp/http.go

```diff
@@ -78,7 +78,7 @@ type Config struct {
 	modules.ModuleConfig
 	Port    int           `yaml:"port"`
 	Timeout time.Duration `yaml:"timeout"`
-	Debug   *bool         `yaml:"debug"`
+	Debug   bool          `yaml:"debug" default:"true"`
```

Comment:
#269 i can remove that `if !cv.HasValue{...}` so default tag can be populate when value is empty